api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -147,4 +148,23 @@ message LocalRateLimit {
   // of the default ``UNAVAILABLE`` gRPC code for a rate limited gRPC call. The
   // HTTP code will be 200 for a gRPC response.
   bool rate_limited_as_resource_exhausted = 15;
+
+  // Rate limit configuration that used to generate a list of description entries based on
+  // the request context. And the generated entries will be used to find one or multiple matched rate
+  // limit rule from the ``descriptors``.
+  // If this is set, then
+  // :ref:`VirtualHost.rate_limits<envoy_v3_api_field_config.route.v3.VirtualHost.rate_limits>` or
+  // :ref:`RouteAction.rate_limits<envoy_v3_api_field_config.route.v3.RouteAction.rate_limits>` fields
```

Comment:
This should probably be mentioned in those fields as well.

Reply:
Check the fields documents again, the legacy `rate_limits` in the route may be shared by multiple different rate limit filters. Additional comments may bring confusions to the users.

For users who doesn't use this new `rate_limits` here, nothing is changed.
For users who use this new `rate_limits` here, they will get the new behavior clearly according to the comments here.

Reply:
Maybe it will be possible to add a warning when there are fields that are ignored.
I know it doesn't add much, but as it stands, an operator that attempts to debug why some route isn't working as they are expecting, needs to go through different levels of route settings, and looking at different places in the config.

Reply:
> Maybe it will be possible to add a warning when there are fields that are ignored.

Sorry, could you make this a little more clear? should I add warning in the runtime log or the comments?

Reply:
> Sorry, could you make this a little more clear? should I add warning in the runtime log or the comments?

I meant the log. The example I'm thinking of is that if the config contains both the new rate-limit filter config and VirtualHost.rate_limits (for example), then Envoy will just warn that the VirtualHost.rate_limits is ignored in favor of the filter config.